taxi/views.py

```diff
@@ -79,11 +88,62 @@ class CarDeleteView(LoginRequiredMixin, generic.DeleteView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    fields = [
+        "username",
+        "first_name",
+        "last_name",
+        "license_number",
+        "password"
+    ]
+    template_name = "taxi/driver_form.html"
+    success_url = reverse_lazy("taxi:driver-list")
+
+    def form_valid(self, form):
+        form.instance.set_password(form.cleaned_data["password"])
+        return super().form_valid(form)
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+    template_name = "taxi/driver_confirm_delete.html"
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/driver_license_update_form.html"
+    success_url = reverse_lazy("taxi:driver-detail")
+
+    def get_success_url(self):
```

Comment:
Here you don't need to set get_success_url because get_absolute_url is defined in your model - this will redirect to DriverDetail by itself